Automattic/jetpack
# Social Menu & Social Media Icons: Add Telegram

Fixes #16281

#### Changes proposed in this Pull Request:

This PR: 
- Adds a new Telegram SVG from automattic/social-logos
- Starts using that SVG in both the widget and the Social Menu

#### Jetpack product discussion

* No

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

Start with a site that runs a theme such as [Floral](https://wordpress.com/theme/floral)

* Go to Jetpack > Settings, and enable the Widgets feature
* Go to Appearance > Customize > Widgets, and add a Social Icons widget to your site.
* Paste a link to a Telegram profile, e.g. t.me/nickname or telegram.me/nickname
* Ensure that the widget uses a Telegram logo.
* In Appearance > Customize > Menu, add a social menu using a similar link, then verify the icon displays as expected.

#### Proposed changelog entry for your changes:

* Social Menu & Social Media Icons: add support for Telegram profiles.
